libs/voicefocus/voicefocus.js

```diff
@@ -292,6 +299,13 @@ class VoiceFocus {
             return node;
         });
     }
+    destroy() {
```

Comment:
Checking how this can be used in our meetingV2 demo? Accompanying demo changes will help in testing and understanding the usage. Also, update to the documentation if needed would also be helpful.

Reply:
It's technically not needed in the meetingV2 demo since it creates a voice focus device and the device is stopped when the participant leaves (thus cleaning up the worker): https://github.com/aws/amazon-chime-sdk-js/blob/main/demos/browser/app/meetingV2/meetingV2.ts#L1826.

We can potentially use the destroy by adding it to the transformer first and then calling it here https://github.com/aws/amazon-chime-sdk-js/blob/7428c620fa0233d5c8e88587cea5bbd2068a1409/demos/browser/app/meetingV2/meetingV2.ts#L480-L483 in the case that VF is not supported.

I'll wait for @michhyun1 to  give this a try and verify the fix before making those changes though.